### ISSUE TITLE
Export AccessFeatureType for paths

### ISSUE THREAD

**Issue by Robbendebiene** [OPENER-next/osm2vdv462#6]
AccessFeatureType
```
lift
escalator
ramp
stairs
```

On first glance this should be rather trivial. Just an aggregate function across all elements (their tags) that looks for the respective highway tags like stepts, elevator etc.

Problem: A path might contain multiple `AccessFeatureType`. This is because we create `StopPlaces` mostly without `AccessSpaces` which are used for:

> Eingangshalle, Zwischengeschoss oder Korridor innerhalb eines Umsteigebauwerks

So there are two options:
- Expect that `AccessSpaces` for every mezzanine exist in OSM
- Generate `AccessSpaces` for mezzanines (when a path goes across multiple levels, we need in between `AccessSpaces` ) Problem however is, that these require unique DHIDs at least by the specification.

**Comment by Robbendebiene** [comment]
Idea 1:

Add "fake osm tags" to the path link export "tags" column.
stairs:
```
highway=steps
```
escelator:
```
highway=steps
conveying=yes/forward/backward/reversible
```
lift:
```
highway=elevator
```
ramp:
```
highway=footway
incline=*
```

Idea 2 (preferred):

Use PathElementsRef table to JOIN imported OSM elements with PathLinks. Then use all the tags there to extract VDV462 export data.

Question: Will the join for these special path links (escalator, lift, stairs) always result in a single "match"? Or can we for example accidently also match a connecting way to a path link that is a stair.

**Comment by janikkaden** [comment]
Idea 2 is implemented in commit [#87fdb83](https://github.com/OPENER-next/osm2vdv462/commit/87fdb830eb4c138483af936883d0db34be7926ac)

Only one osm element is matched to one path link to extract the tags and derive the access feature type.
